src/main/resources/META-INF/fabric8/profiles.yml

```diff
@@ -0,0 +1,60 @@
+- name: istio-default
+  enricher:
+    includes:
+    - fmp-name
+    - fmp-controller
+    - fmp-service
+    - fmp-image
+    - fmp-portname
+    - fmp-ianaservice
+    - fmp-project
+    - fmp-dependency
+    - fmp-pod-annotations
+    - fmp-git
+    # TODO: Documents and verify enrichers below
+    - fmp-debug
+    - fmp-merge
+    - fmp-remove-build-annotations
+    - fmp-volume-permission
+    # -----------------------------------------
+    - f8-cd
+    - f8-cd-doc-link
+    - f8-cd-grafana-link
+    - f8-icon
+    # TODO: Document and verify enrichers below
+    - f8-expose
+    # Istio sidecar injection
+    - fmp-istio-enricher
+    # Health checks
+    - spring-boot-health-check
+    - wildfly-swarm-health-check
+    - karaf-health-check
+    - vertx-health-check
+    - docker-health-check
+    - f8-prometheus
+    - f8-maven-scm
+    - f8-maven-issue-mgmt
+    # Dependencies shouldn't be enriched anymore, therefor it's last in the list
+    - fmp-dependency
+    - f8-watch
+    - fmp-revision-history
+    - fmp-docker-registry-secret
+    excludes:
+    - fmp-openshift-route
+    config:
+      fmp-istio-enricher:
```

Comment:
Looks good but I'm wondering if/how this config can be overridden in projects using the enricher. This needs to be checked before merging.

Reply:
You can override them in plugin configuration:
```
         <plugin>
            <groupId>io.fabric8</groupId>
            <artifactId>fabric8-maven-plugin</artifactId>
            <configuration>
              <profile>istio-default</profile>
              <enricher>
                <config>
                  <fmp-istio-enricher>
                    <istioVersion>...</istioVersion>
                  </fmp-istio-enricher>
                </config>
              </enricher>
            </configuration>
          </plugin>
```

Reply:
👍

Reply:
Is there a reason why the default istio version is 0.4.0? If not, I'd suggest using the latest version.

Reply:
This I'm not exactly sure. 0.4.0 was in the previous examples of this enricher.

Which version are you currently using with the new boosters? I've tested mutual tsl on 0.4.0, but Libor is getting some issues with 0.6.0 on it.

Reply:
> 0.4.0

No but that was the version available when the enricher has been created.

If you target to use `0.6.0`, please verify like also for `0.5.0` that the args injected by ``istioctl inject ...` command are still the same. If this is not the case, then the class where such arguments are defined should be updated

Reply:
So it seems that we should plan a separate task which is to upgrade the enricher to work with the newer versions.
I think we can keep 0.4.0 in the profiles.yml and update it once the whole enricher is verified to work with the newer version.

Reply:
That would be a good opportunity to add some tests too.

Reply:
I will work on the enricher.